docs/admin-guide/load-config.md

```diff
@@ -1,125 +1,92 @@
-# The load config
+# The Load Config File
+
 
 Scout have the possibility to store loads of information about a case and the samples that are included. It is cumbersome to specify to many parameters on the command line so there is an option to give this information in a yaml formated config file.
```

Comment:
This part has many typos, what about fixing it into something:

```
A case and its individuals (or samples, in cancer track) can be uploaded into Scout using specially formatted .yaml config files. Config files contain information about the analysis, panels used, path to VCF and eventual alignment files and more.
The following template illustrates the basic structure of the .yaml config file:
```

Reply:
Done